src/main/java/org/galatea/starter/service/HalService.java

```diff
@@ -32,7 +44,10 @@
    * @return the result of executing the command with the given parameters
    */
   public String processText(String text) {
-    switch (text) {
+    /* Send raw text to wit.ai */
+    WitResponse witResponse = witGetter.getWitResponse(text);
+
+    switch (witResponse.getEntities().getIntent()[0].getValue().toString()) {
```

Comment:
Do we need to add some checks to the validity of the Wit response here? 

Also, if the highest confidence option is less than, say, 0.5, should we just refuse to process the message and respond with a cannot understand the command error?